patterns/1-initial/modular-code.md

```diff
@@ -24,33 +24,50 @@ Development does not want to spend the extra resources needed to develop modular
 * Extra documentation is required for reusable code.
 * Some companies have a common components group that develops reusable code, but others feel that such components should be developed by those business lines that are using the components and a library of common components could be established.
 * Developers might not know how to write modular code. Education might be needed.
+
 * Might be a fear that if not done properly, quality might be impacted.
 * Developers might not have incentive to write modular code (due to their tight schedules and lack of a mandate).
 * You might be dealing with legacy systems (can't be simply refactored or rewritten).
 * Requirements might be different for writing modular code.
 * Architectural constraints might impact modularity.
 * Developers who develop monolithic code bases might lack the perspective of how modularity might improve how they work.
+* If there is frequent turnover of team members, modularization may not be a priority.
```

Comment:
Just to clarify: modularization might not be a **priority for developers,** if they think they are going to leave the team, soon? I would argue it's definitively a priority from an organisations perspective, because modular code should be more understandable and maintainable and that in turn helps when new team members are onboarded.

Reply:
WDYT, @MaineC and @NewMexicoKid ?